node/rpc/src/types.rs

```diff
@@ -23,10 +23,29 @@ pub struct StakeRequest {
 pub struct DeployContractRequest {
     pub nonce: u64,
     #[serde(with = "bs58_format")]
-    pub owner_account_id: AccountId,
-    #[serde(with = "bs58_format")]
     pub contract_account_id: AccountId,
     pub wasm_byte_array: Vec<u8>,
+    pub public_key: Vec<u8>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct CreateAccountRequest {
```

Comment:
again, i would be more verbose here rather than just reusing the transaction struct